src/convergence_factors.jl

```diff
@@ -0,0 +1,145 @@
+import Dates
+import SciMLBase
+import ClimaComms
+import ClimaCore as CC
+import ClimaTimeSteppers as CTS
+import ClimaCoupler:
+    Checkpointer,
+    FieldExchanger,
+    FluxCalculator,
+    Interfacer,
+    TimeManager,
+    Utilities
+
+function compute_actual_rho(physical_values)
```

Comment:
"actual_rho" is a bit strange naming. maybe `compute_ρ_analytical` or `compute_ρ_cont`? Or similar?